.github/workflows/workflow.yaml

```diff
@@ -9,14 +9,14 @@ env:
 
 jobs:
   flake8:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-23.04
```

Comment:
I think it's generally safer (in the sense of "principle of least surprise") to use LTS versions?